.gitignore

```
@@ -105,3 +105,4 @@ venv.bak/
 
 # IntelliJ
 .idea
+Pipfile*
```

@avallbona could you add a line at the end of the file?

```suggestion
Pipfile*

```

---

done